.github/workflows/unittests.yml

```diff
@@ -141,14 +141,22 @@ jobs:
           sudo make install
           cd ../../java
           mvn clean install -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -P arrow-jni -am -Darrow.cpp.build.dir=/tmp/arrow/cpp/build/release/ -DskipTests -Dcheckstyle.skip
+      - name: Install PySpark dependencies
+        run: |
+          sudo apt-get install python3.6
+          python3 --version
+          pip3 install pyspark
+          pyspark --version
+          pip3 install pyarrow
       - name: Run unit tests
         run: |
           mvn clean install -N
           cd arrow-data-source
           mvn clean install -DskipTests -Dbuild_arrow=OFF
           cd ..
           mvn clean package -P full-scala-compiler -Phadoop3.2 -am -pl native-sql-engine/core -DskipTests -Dbuild_arrow=OFF
```

Comment:
it seems the profile is wrong - should be `hadoop-3.2`
https://github.com/oap-project/gazelle_plugin/blob/master/pom.xml#L66

Reply:
fixed here: https://github.com/oap-project/gazelle_plugin/pull/393/files#diff-54c039feb42ef7c758e4cbc3aaa48971a1b41a1132399420b4c98d984ad62828R150